pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/metrics/xml/CyclomaticComplexity.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<test-data>
+<!--
```

Comment:
Many test cases are still commented out. Are these work-in-progress or can we already enable them?